kurtosis-module/impl/participant_network/mev_boost/mev_boost_launcher.go

```diff
@@ -1,18 +1,61 @@
 package mev_boost
 
-import "github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/enclaves"
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/enclaves"
+	"github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/services"
+	"github.com/kurtosis-tech/stacktrace"
+)
 
 type MEVBoostLauncher struct {
-	// TODO Analogous to GethLauncher, BesuLauncher, NimbusLauncher, etc.
+	RelayCheck bool
+	Relays     []string
 }
 
-/*
-KEVIN NOTE: You don't need this to be an interface yet because you only have one mev-boost node (the flashbots one),
-but if you got multiple in the future you'd probably want a MEVBoostLauncher interface with a Launch function, and
-then have implementations of it
- */
-func (launcher *MEVBoostLauncher) Launch(enclaveCtx enclaves.EnclaveContext) (*MEVBoostContext, error) {
-	// TODO A function for launching a MEV Boost node in the same way that GethLauncher
-	//  In the future, this will probably take in relay information as well
-	panic("TODO IMPLEMENT ME")
+const (
+	flashbotsMevBoostImage        = "flashbots/mev-boost"
+	flashbotsMevBoostPort  uint16 = 18550
+)
+
+var (
+	usedPorts = map[string]*services.PortSpec{
+		"api": services.NewPortSpec(flashbotsMevBoostPort, services.PortProtocol_TCP),
+	}
+	networkIdToName = map[string]string{
+		"5":        "goerli",
+		"11155111": "sepolia",
+		"3":        "ropsten",
+	}
+)
+
+func (launcher *MEVBoostLauncher) Launch(enclaveCtx *enclaves.EnclaveContext, serviceId services.ServiceID, networkId string) (*MEVBoostContext, error) {
+	containerConfigSupplier := func(string) (*services.ContainerConfig, error) {
+		relayCheckOption := ""
+		if launcher.RelayCheck {
+			relayCheckOption = "-relay-check"
+		}
+		networkName, ok := networkIdToName[networkId]
+		if !ok {
+			networkName = fmt.Sprintf("network-%s", networkId)
+		}
+		command := []string{
+			"mev-boost",
+			fmt.Sprintf("-%s", networkName),
+			relayCheckOption,
+			fmt.Sprintf("-relays %s", strings.Join(launcher.Relays, "")),
+		}
+		containerConfig := services.NewContainerConfigBuilder(flashbotsMevBoostImage).WithUsedPorts(usedPorts).WithCmdOverride(command).Build()
+		return containerConfig, nil
+	}
+
+	serviceCtx, err := enclaveCtx.AddService(serviceId, containerConfigSupplier)
+	if err != nil {
+		return nil, stacktrace.Propagate(err, "An error occurred launching the Geth EL client with service ID '%v'", serviceId)
+	}
+
+	return &MEVBoostContext{
```

Comment:
Per the comment above, this is where I'd construct the endpoint URL and call `NewMEVBoostContext`. The benefit of doing it here is the `"api"` port is encapsulated completely inside this file, so `MEVBoostContext` becomes an interface for providing information about the MEV boost node without the user of MEVBoostContext needing to know about how that information was retrieved.